### DISCUSSION TITLE
Any plans for authentication/authorisation support in Marquez?

### DISCUSSION THREAD

**Discussion by laurent-opnworks** [OpenLineage/OpenLineage#2204] [General]
Hi all,

Anybody working on this? Authorisaton should be at the namesapce level I presume.

I realize this can be a realtively major undertaking.

**Comment by pawel-big-lebowski** [comment]
Hi @laurent-opnworks, 

You can try asking this on Marquez Github https://github.com/MarquezProject/marquez 
@wslulciuc do you happen to know if anyone is working on that?
